src/main/java/entities/DesignableMaze.java

```diff
@@ -1,7 +1,14 @@
 package entities;
```

Comment:
please overwrite this entity file with what is on main. 

https://github.com/CSC207-2022F-UofT/course-project-wasted-potential/blob/main/src/main/java/entities/DesignableMaze.java